lib/Extractor/Candidates.cpp

```diff
@@ -489,7 +489,8 @@ Inst *ExprBuilder::build(Value *V) {
           return IC.getInst(Inst::UMulWithOverflow, L->Width+1, {Mul, Overflow});
         }
       }
-    } else if (TLI->getLibFunc(*Call->getCalledFunction(), Func) && TLI->has(Func)) {
+    } else if (Call->getCalledFunction() &&
```

Comment:
do you want to put the pointer into a variable install of calling getCalledFunction() twice?

Reply:
I tried to do initialization and boolean calculation in a single if statement to stop bringing new blocks, but it is not possible. Anyway, a new patch is submitted.